solr/core/src/java/org/apache/solr/handler/admin/ClusterStatus.java

```diff
@@ -116,9 +118,19 @@ public void getClusterStatus(NamedList<Object> results)
       // add live_nodes
       if (withLiveNodes) clusterStatus.add("live_nodes", liveNodes);
     }
+
+    Aliases aliases = null;
+    if (withCollection || withAliases) {
```

Comment:
why withCollection?

Reply:
Whether a collection parameter is passed (where the input could be either a collection or an alias) or if includeAll is set to true, aliases are retrieved from ZkStateReader to create a mapping of collections to the list of aliases associated with them and returned as part of the response.

Reply:
Since they occupy the same namespace, I could see returning aliases if you want collections.